docs/pages/setup/reference/cli.mdx

```diff
@@ -686,11 +687,14 @@ registered devices using [`tsh mfa ls`](cli.mdx#tsh-mfa-ls).
 $ tsh mfa rm <device-name>
 ```
 
+### tsh environment variables
```

Comment:
I think this header was missing (without it the variables are all inside the "tsh mfa add" header).